recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/icepyx-{{ version }}.tar.gz
-  sha256: b63ba2041377338e692669d57b8b4aa6659fe15fff482e8006022887d6c4a703
+  sha256: f0ecc8136f306e0e11f795fa5e26a7211f6f5682d88d14686fa41dc80efaf5a2
```

Comment:
Might need to tag and create an `icepyx-v0.6.0.post0` release on PyPI that includes the https://github.com/icesat2py/icepyx/pull/287 bugfix. I would suggest yanking the v0.6.0 release from PyPI too since it doesn't actually work.

Reply:
Thanks for the suggestion - I've yanked 0.6.0 (moving the tag to the non-working version) and released 0.6.1 with the bug fix.